projects/vdk-plugins/vdk-impala/src/vdk/plugin/impala/impala_error_handler.py

```diff
@@ -7,6 +7,8 @@
 from vdk.internal.builtin_plugins.connection.recovery_cursor import RecoveryCursor
 from vdk.internal.core import errors
 
+MEMORY_LIMIT_PATTERN = r"Limit=(\d+\.\d+)\s*([KMGTP]B)"
```

Comment:
TB and PB , Considering that is memory limit per node , that would be impressive query :) 

It doesn't hurt to have. Thinking about the future